docs/reference/connectivity.md

```diff
@@ -15,3 +15,43 @@ Ella Core uses 4 different interfaces:
   ![Connectivity](../images/connectivity.svg){ width="800" }
   <figcaption>Connectivity in Ella Core</figcaption>
 </figure>
+
+# Combining interfaces
+
+It is possible to combine interfaces in the following manners.
+
+## Combined N2 and N3
+
+Many gNodeBs can use a single network link towards the core. In this case,
+N2 and N3 can be combined by using the same interface name for both of them in the
+configuration file.
+
+<figure markdown="span">
+  ![Combined N2/N3](../images/combined_n2_n3.svg){ width="800" }
+  <figcaption>Combined N2 and N3</figcaption>
+</figure>
+
+## Combined API and N6
+
+The API interface is often the management interface with internet access,
+and the N6 interface also requires internet access. They can be combined by using
+the same interface name for both in the configuration file.
+
+<figure markdown="span">
+  ![Combined API/N6](../images/combined_api_n6.svg){ width="800" }
+  <figcaption>Combined API and N6</figcaption>
+</figure>
+
+## Minimal interface requirements
```

Comment:
We already have a page for system requirements. If we have more requirements we should probably define them over there? Here we can simply add another sub header for "Combined API/N6 and combined N2/N3".

Reply:
Agreed, I'll change the sub header.